src/App.js

```diff
@@ -1,13 +1,139 @@
-import React from 'react';
+import React from "react";
+import TodoList from "../src/components/TodoComponents/TodoList";
+import TodoForm from "../src/components/TodoComponents/TodoForm";
+import Img from '../src/components/TodoComponents/Img';
+import Music from '../src/components/Music';
 
 class App extends React.Component {
   // you will need a place to store your state in this component.
   // design `App` to be the parent component of your application.
   // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor() {
+    super();
+
+    this.state = {
+      play: false,
+      pause: true,
+      search: "",
+      todo: "",
+      todos: []
+    };
+  }
+
+  inputChange = event => {
+    this.setState({
+      todo: event.target.value
+    });
+
+    localStorage.setItem('todo', JSON.stringify(event.target.value));
+  };
+
+  addTodo = event => {
+    event.preventDefault();
+    this.setState({
+        todos: [
+          ...this.state.todos,
+          {
+            task: this.state.todo,
+            id: Date.now(),
+            completed: false
+          }
+        ],
+        todo: ""
+    });
+    localStorage.setItem('todos', JSON.stringify(this.state.todos));
+    localStorage.setItem('todo', '');
+  };
+
+  clear = (event, todoComplete) => {
+    event.preventDefault();
+    const notDone = this.state.todos.filter(todo => todo.completed === false);
+    console.log(notDone);
+    this.setState({
+      todos: notDone
+    });
+    localStorage.setItem('todos', JSON.stringify(notDone));
+  };
+
+  clickHandler = todoId => {
+    this.setState({
+      todos: this.state.todos.map(todo => {
+        if (todo.id === todoId) {
+          return {
+            task: todo.task,
+            id: todo.id,
+            completed: !todo.completed
+          };
+        } else {
+          return todo;
+        }
+      })
+    });
+  };
+
+  searchFunc = (event) => {
```

Comment:
I think calling this method `search` would be more consistent with your naming of the `reload` & `play` functions below. You could